pandas/io/tests/test_pytables.py

```diff
@@ -4623,6 +4624,27 @@ def _test_sort(obj):
     else:
         raise ValueError('type not supported here')
 
+class TestToHdfWithIntegerColumnNames(tm.TestCase):
+    # GH9057
+    def setUp(self):
+        N = 2
+        array = np.random.randint(0,8, size=N*N).astype('uint8').reshape(N,-1)
+        df = DataFrame(array, index=pd.date_range('20130206',
+                                                  periods=N,freq='ms'))
+
+        self.filename = NamedTemporaryFile(suffix='.h5', delete=False).name
+        df.to_hdf(self.filename,'df', mode='w', format='table',
+                  data_columns=True)
+
+    def test_file_is_openable(self):
+
+        with tables.open_file(self.filename, 'r') as myfile:
+            stuff = myfile.root.df
```

Comment:
test that the read back works with regular selection (look at other tests for how to do this)
further need a test with a query, which uses a stringified column